src/components/kyb/FinancialInformationStep.tsx

```
@@ -0,0 +1,239 @@
+        if (!value || value.trim().length < 10) {
+          newErrors.businessPurpose = 'Business purpose must be at least 10 characters';
+        } else {
+          delete newErrors.businessPurpose;
+        }
+        break;
+    }
+
+    setErrors(newErrors);
+  };
+
+  const validateAllFields = () => {
```

ESLint

[error] 'validateAllFields' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)